docs/v2-system-addon/data_dictionary.md

```diff
@@ -167,10 +201,12 @@ and losing focus. | :one:
 | `value` | [Required] An integer that represents the measured performance value. Can store counts, times in milliseconds, and should always be a positive integer.| :one:
 | `ver` | [Auto populated by Onyx] The version of the Onyx API the ping was sent to. | :one:
 | `highlights_size` | [Optional] The size of the Highlights set. | :one:
+| `highlights_missing_data`| [Optional] Difference between data availability timestamp and first call to component render function. | :one:
```

Comment:
You don't need to document the values for the `event` key here, this is only for the `keys` in the ping.

Reply:
@ncloudioj so where does that stuff get documented?

Reply:
You can document this under the corresponding sections in the `data_event.md`.